src/Model/Table/SentencesTable.php

```diff
@@ -278,6 +278,10 @@ public function afterSave($event, $entity, $options = array())
         ));
         $this->getEventManager()->dispatch($event);
 
+        if (!is_null($entity->user_id)) {
+            $this->Users->updateLastContribution($entity->user_id);
```

Comment:
There is a logic problem here. If a corpus maintainer or an admin edits someone else’s sentence, it’s gonna update the last_contribution time of the sentence owner instead of the actual sentence editor.